python/delta_sharing/delta_sharing.py

```diff
@@ -49,14 +49,19 @@ def _parse_url(url: str) -> Tuple[str, str, str, str]:
     return (profile, share, schema, table)
 
 
-def load_as_pandas(url: str, limit: Optional[int] = None) -> pd.DataFrame:
+def load_as_pandas(
+    url: str,
+    limit: Optional[int] = None,
+    version_of: Optional[int] = None
+) -> pd.DataFrame:
     """
     Load the shared table using the give url as a pandas DataFrame.
 
     :param url: a url under the format "<profile>#<share>.<schema>.<table>"
     :param limit: a non-negative int. Load only the ``limit`` rows if the parameter is specified.
       Use this optional parameter to explore the shared table without loading the entire table to
       the memory.
+    :param version_of: a optional non-negative in. Load the snapshot of table at version_of
```

Comment:
updated to version.
Was using version_of because there was a `version` variable in the spark code, so I have to come up with a different name. It's not a problem in the python code.